autopush-common/Cargo.toml

```
@@ -62,13 +62,15 @@ prometheus-client = { version = "0.23", optional = true }
 protobuf = { version = "=2.28.0", optional = true } # grpcio does not support protobuf 3+
 form_urlencoded = { version = "1.2", optional = true }
 
-redis = { version = "0.29", optional = true }
+deadpool-redis = { version = "0.20", optional = true }
+redis-async = { version = "0.17", optional = true }
 redis-module = { version = "2.0", optional = true }
+redis = { version = "0.29", optional = true }
```

this crate isn't used, the redis async stuff is with that crate's `aio` feature -- might want to specify it here just to be clear (`deadpool-redis specifies it`)
```suggestion
deadpool-redis = { version = "0.20", optional = true }
redis-module = { version = "2.0", optional = true }
redis = { version = "0.29", optional = true }
```